sources/packages/web/src/views/student/application-offering-change-request/tabs/RequestedApplicationOfferingDetails.vue

```diff
@@ -80,17 +80,17 @@ export default defineComponent({
           Location: changeRequest.value.locationName,
         } as Record<string, string>),
     );
-    const changeRequestNotApproved = computed(() => {
+    const changeRequestInProgressWithStudent = computed(() => {
```

Comment:
Just for my clarification because I was not part of previous reviews, I can see on Figma that these texts are not displayed for "Approved" status. Why are we showing them only for `InProgressWithStudent`? Are they not supposed to be hidden only for `Approved` ones?

Reply:
In the story AC, the requirement is that the "Text under the Application Details and Request Details headings only need to be displayed when "In progress with student." They can be omitted after student approves or declines."